vault_core/src/vault_staker_withdrawal_ticket.rs

```diff
@@ -28,6 +28,11 @@ pub struct VaultStakerWithdrawalTicket {
     /// This is used to ensure the amount redeemed is the same as the amount allocated.
     vrt_amount: PodU64,
 
+    /// The minimum supported mint to be withdrawn when this ticket is burned
+    /// This is required to factor in a 0.5% slippage of the current exchange
```

Comment:
current exchange -> exchange rate at time of ticket creation

Reply:
10-4